index.html

```diff
@@ -1,17 +1,55 @@
-Katsiaryna Lashcheuskaya
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Fenrir Intro Project</title>
+    <meta charset="UTF-8">
+    <meta name="keywords" content="HTML, CSS, JavaScript">
+    <meta name="author" content="Katsiaryna Lashcheuskaya">
+</head>
+<body>
 
-About
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
-Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
-Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+<h1>Katsiaryna Lashcheuskaya</h1>
 
-Experience
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
-Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
-Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+<section>
+    <h1>About</h1>
+    <p>Hi there, I'm a frontend developer. I have experience in creating SPA using React, Redux, TypeScript. I'm continuously improving my English,
+        studying new technologies and solving tasks on Codewars in my spare time. Also, my strengths are great analytical skills,
+        responsibility and stress resistance.
+    </p>
+</section>
 
-Connect
-ek.sharova@icloud.com
-https://github.com/katsiarynalashcheuskaya
+<section>
+    <h1>Skills</h1>
+    <ul>
+        <li>HTML/CSS/SCSS</li>
+        <li>JavaScript/TypeScript</li>
+        <li>React/Redux/Redux Toolkit</li>
+        <li>Postman</li>
+        <li>Axios/REST API</li>
+        <li>Git/GitHub</li>
+        <li>Unit-test/Storybook</li>
+        <li>Material UI</li>
+        <li>SASS</li>
+        <li>Figma</li>
+    </ul>
+</section>
+
+<section>
+    <h1>Experience</h1>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+        magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+        consequat.
+        Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+        Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+    </p>
+</section>
+
+<section>
+    <h1>Connect</h1>
+    <div>
+        <span>ek.sharova@icloud.com</span>
+        <span>https://github.com/katsiarynalashcheuskaya</span>
+    </div>
```

Comment:
These display side-by-side in the browser.  Better to use `<ul>` and `<li>` for each item.